packages/client/src/Tests/mappingHelpers/assetsVisibleToMe.test.js

```diff
@@ -0,0 +1,84 @@
+/* global it expect */
+
+import assetsVisibleToMe from '../../Components/Mapping/helpers/assetsVisibleToMe'
+import { UMPIRE_FORCE } from '../../consts'
+
+/* sample dataset that is used across all tests in this file
+*/
+const allForces = [
+  { name: 'umpire' },
+  {
+    name: 'Blue',
+    assets: [
+      {
+        name: 'alpha',
+        perceptions: {
+          Red: { force: 'Blue', type: 'Frigate' }
+        }
+      },
+      {
+        name: 'bravo',
+        perceptions: {
+        }
+      }
+    ]
+  },
+  {
+    name: 'Red',
+    assets: [
+      {
+        name: 'charlie',
+        perceptions: {
+          Blue: { force: 'Green', type: 'Frigate' }
+        }
+      },
+      {
+        name: 'delta',
+        perceptions: {
+        }
+      }
+    ]
+  },
+  {
+    name: 'Green',
+    assets: [
+      {
+        name: 'echo',
+        perceptions: {
+          Blue: { force: 'Green', type: 'Frigate' },
+          Red: { force: 'Green', type: 'Frigate' }
+        }
+      },
+      {
+        name: 'foxtrot'
+      }
+    ]
+  }
+]
+
+it('I am umpire force', () => {
```

Comment:
This is personal preference but in my head, user stories can start with `I ...` (although I'd still argue for more context there too) but unit tests should again be imperative and also follow an 'it' statement in a way that is lexically correct, e.g.:

`it('correctly returns an umpire force')`

Although that may not be a perfect example is that's not really what it is doing. Perhaps something like:

`it('Returns 6 visible assets to the Umpire force and 'Alpha' should be amongst them')` (or something like that).

Reply:
Aah, I see. Yes, I'll adopt it. I was struggling to recognise the value of the `it` bit.  Thanks.